run-testnet-relay.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Starting relay...${LOG_END}"
+
+cd "$WORKDIR/relays/maintainer/"
+
+# Install python virtualenv.
+pipenv install --python=$(pyenv which python3.7)
```

Comment:
Shouldn't this be in `install-testnet.relay.sh`?

Reply:
Yep, good point!